packages/app/components/messages/message-row.tsx

```diff
@@ -12,13 +13,65 @@ import {
   Message,
 } from "@showtime-xyz/universal.icon";
 import { colors } from "@showtime-xyz/universal.tailwind";
-import { Text, linkify } from "@showtime-xyz/universal.text";
+import { Text } from "@showtime-xyz/universal.text";
 import { VerificationBadge } from "@showtime-xyz/universal.verification-badge";
 import { View } from "@showtime-xyz/universal.view";
 
 import { Link } from "app/navigation/link";
 import { convertUTCDateToLocalDate, formatAddressShort } from "app/utilities";
 
+import { useRouter } from "design-system/router";
```

Comment:
for unity, need to use `import { useRouter } from "@showtime-xyz/universal.router";` replace it.